### ISSUE TITLE
BetterChannelList crashing on specific servers

### ISSUE THREAD

**Issue by Test-Account666** [arg0NNY/DiscordPlugins#109]
### Plugin:

📜 BetterChannelList

### What version of BetterDiscord are you using?

1.10.2

### What version of the selected plugin(s) are you using (if applicable)?

1.1.5

### What client are you using?

🔵 Stable

### Describe the bug

When I click on a specific server, discord crashes.

No idea why this specific one.

It worked before.

Disabling `BetterChannelList` fixes the issue though.

### Additional comments

Error messages I found in the console:

```
[BetterDiscord] [Patcher] Could not fire instead callback of Z for BetterChannelList TypeError: Cannot destructure property 'channel' of 'props' as it is undefined.
    at Object.eval [as callback] (BetterChannelList.plugin.js:938:21)
    at renderer.js:5:22680
    at q (web.35a426710db29dd3edca.js:12:1265828)
    at Q (web.35a426710db29dd3edca.js:12:1266487)
    at og (web.35a426710db29dd3edca.js:12:1331051)
    at lr (web.35a426710db29dd3edca.js:12:1369815)
    at lo (web.35a426710db29dd3edca.js:12:1370583)
    at s4 (web.35a426710db29dd3edca.js:12:1366933)
    at s8 (web.35a426710db29dd3edca.js:12:1367739)
    at iT (web.35a426710db29dd3edca.js:12:1306525)
    at l_ (web.35a426710db29dd3edca.js:12:1372300)
    at ld (web.35a426710db29dd3edca.js:12:1371265)
    at s3 (web.35a426710db29dd3edca.js:12:1366210)
    at v (web.35a426710db29dd3edca.js:12:1555532)
    at b (web.35a426710db29dd3edca.js:12:1556062)
    at T.C.<computed> (web.35a426710db29dd3edca.js:1:1271621)
    at L (web.35a426710db29dd3edca.js:1:1271386)
    at MessagePort.D (web.35a426710db29dd3edca.js:1:1271445)

web.35a426710db29dd3edca.js:12 TypeError: format is not a function
    at buildLastMessageContent (BetterChannelList.plugin.js:343:20)
    at ChannelLastMessage (BetterChannelList.plugin.js:410:32)
    at aY (web.35a426710db29dd3edca.js:12:1322668)
    at l (web.35a426710db29dd3edca.js:12:1379315)
    at lu (web.35a426710db29dd3edca.js:12:1370772)
    at ls (web.35a426710db29dd3edca.js:12:1370700)
    at lo (web.35a426710db29dd3edca.js:12:1370563)
    at s4 (web.35a426710db29dd3edca.js:12:1366933)
    at s8 (web.35a426710db29dd3edca.js:12:1367739)
    at iT (web.35a426710db29dd3edca.js:12:1306525)
    at l_ (web.35a426710db29dd3edca.js:12:1372300)
    at ld (web.35a426710db29dd3edca.js:12:1371265)
    at s3 (web.35a426710db29dd3edca.js:12:1366210)
    at v (web.35a426710db29dd3edca.js:12:1555532)
    at b (web.35a426710db29dd3edca.js:12:1556062)
    at T.C.<computed> (web.35a426710db29dd3edca.js:1:1271621)
    at L (web.35a426710db29dd3edca.js:1:1271386)
    at MessagePort.D (web.35a426710db29dd3edca.js:1:1271445)
    ```

### I have read and agree to the following:

- [X] I have searched existing/closed issues to confirm that this hasn't been solved already.
- [ ] I've already tried to disable all other plugins (or themes) to confirm this issue is related to the plugin.
This isn't necessarily required

**Comment by Test-Account666** [comment]
More information:
Apparently, this only happens, if I got pinged in that server

**Comment by ThatXliner** [comment]
<img width="1705" alt="image" src="https://github.com/user-attachments/assets/638f6d35-1e9e-43b6-ab83-fd4fc32d7e94">



> More information:
> Apparently, this only happens, if I got pinged in that server

or maybe when a ping appears. like it crashes on this server: https://discord.gg/K5nzgVKup2

**Comment by arg0NNY** [comment]
Fixed in cedca17bdddc43f7a20343855758042124f0dd78
